lib/App/index.js

```diff
@@ -209,7 +209,7 @@ class App {
           const isAppCapability = (typeof appJson.capabilities !== 'undefined' && typeof appJson.capabilities[capabilityId] !== 'undefined');
 
           if (!isSystemCapability && !isAppCapability) {
-            throw new Error(`Invalid capability: ${capability}`);
+            throw new Error(`drivers.${driver.id} invalid capability: ${capability}`);
```

Comment:
Couldn't figure out a message style I was completely happy with.
This change breaks with the consistent "Invalid ..." elsewhere but does include the driver id which I think is important

Reply:
Invalid capability "${capability}" for driver "${driver.id}"